src/components/ToastCard/ToastCard.tsx

```diff
@@ -0,0 +1,32 @@
+import "./_toast-card.scss";
+
+type Props = {
+  type: string;
+  message: string;
+};
+
+export default function ToastCard({ type, message }: Props) {
+  let iconName;
```

Comment:
Can you add a default here.

Reply:
You could also use an enum (object) which would return the correct value for the class below....a lot less lines :)